src/validation/fields/helper/github-handles.validation.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { ErrorMessages } from '../../error.messages';
+
+export const validateGithubHandles = (value: string) => {
+    if (!value) {
+        throw new Error(ErrorMessages.GIT_HANDLE);
+    }
+
+    const handles = value.split(',').map(handle => handle.trim());
+    const githubHandleRegex = /^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*$/;
```

Comment:
I found a supposedly more robust regex for GitHub handles from [this library](https://www.npmjs.com/package/github-username-regex). 

 `const githubHandleRegex = /^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$/i`

Using it passes your tests, but I would suggest to add more tests / more specific test descriptions to cover these specific criteria as well:

- Github username may only contain alphanumeric characters or hyphens.
- Github username cannot have multiple consecutive hyphens.
- Github username cannot begin or end with a hyphen.
- Maximum is 39 characters.